cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/bungee/command/CommandHub.java

```diff
@@ -42,6 +42,6 @@ public void execute(CommandSender sender, String[] args) {
 
     @Override
     public String[] getAliases() {
-        return new String[]{"lobby", "l", "leave"};
+        return BridgeConfigurationProvider.load().getLobbyCommandAliases().toArray(new String[0]);
```

Comment:
Maybe save this as a array inside of the class instead of re-creating it every time